packages/flutter/lib/src/material/chip.dart

```diff
@@ -1227,6 +1229,7 @@ class ActionChip extends StatelessWidget implements ChipAttributes, TappableChip
     this.elevation,
     this.shadowColor,
   }) : assert(label != null),
+       assert(clipBehavior != null),
```

Comment:
This and similar is a breaking change. I think we should do it, but we should announce it first.